x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action_ess.ts

```diff
@@ -137,6 +140,15 @@ export default ({ getService }: FtrProviderContext): void => {
       expect(sidecarActionsResults.hits.hits.length).toBe(1);
       expect(sidecarActionsResults.hits.hits[0]?._source?.references[0].id).toBe(rule.id);
 
+      const { indexListOfDocuments } = dataGeneratorFactory({
+        es,
+        index: 'logs-1',
+        log,
+      });
+      await indexListOfDocuments([
+        { id: uuidv4(), '@timestamp': moment(new Date()).subtract(5, 's') },
+      ]);
```

Comment:
How does this relate to bulk actions for rules?

Reply:
I think technically this change isn't needed anymore as part of this PR, I've made the index I'm creating hidden so it shouldn't match the `*` pattern this test was using.

But it's probably a good idea to use a more narrow pattern in this test, so I suppose the rest of this code just creates an index that matches the pattern and puts a document in it for the rule to work?

cc @maximpn

Reply:
Okay, since it is not needed anymore, let's roll it back 👍

Reply:
Reverted https://github.com/elastic/kibana/pull/201470/commits/2b7c884ac6a182a0de664a5364246d150c4dd6aa